.gitignore

```
@@ -0,0 +1,7 @@
+work/
+.nextflow
+.nextflow.log*
+
+*fastq.gz
```

I guess we could add fq.gz and .fq as well just to cover all bases
```suggestion
*fq.gz
*fq
```